docs/source/guide/tutorials/base/how_to_train/action_classification.rst

```diff
@@ -0,0 +1,258 @@
+Action Classification model
+================================
+
+This live example shows how to easily train, validate, optimize and export classification model on the `HMDB51 <https://serre-lab.clps.brown.edu/resource/hmdb-a-large-human-motion-database/>`_.
```

Comment:
How about converting `This live example` --> `This documentations` ?

Reply:
I prefer keeping "This live example" or "This tutorial"